src/assets/src/components/common.tsx

```diff
@@ -441,29 +441,54 @@ export const LoginDialog = (props: LoginDialogProps) =>
     </Modal>
 
 
-interface BlueJeansOneTouchDialLinkProps {
+interface OneTouchDialLinkProps {
     phone: string; // "." delimited
-    meetingNumber: string;
+    meetingNumber?: string;
 }
 
-export const BlueJeansOneTouchDialLink = (props: BlueJeansOneTouchDialLinkProps) => 
+const OneTouchDialLink = (props: OneTouchDialLinkProps) => (
     <a href={`tel:${props.phone.replace(".", "")},,,${props.meetingNumber},%23,%23`}>
         {props.phone}
     </a>
+);
 
-interface BlueJeansDialInMessageProps {
-    meetingNumber: string;
+interface IntlTelephoneLinkProps {
+    intlNumbersURL: string;
 }
 
-export const BlueJeansDialInMessage = (props: BlueJeansDialInMessageProps) => {
-    const phoneLinkUsa = <BlueJeansOneTouchDialLink phone="1.312.216.0325" meetingNumber={props.meetingNumber} />
+const IntlTelephoneLink = (props: IntlTelephoneLinkProps) =>  {
+    return (
+        <a target="_blank" href={props.intlNumbersURL}>
+            find your international number to call in from outside the USA
+        </a>
+    );
+}
+
+export type DialInMessageProps = OneTouchDialLinkProps & IntlTelephoneLinkProps;
+const meetingNumberPlaceholder = 'the meeting number (that will appear here once the meeting is created) followed by ';
+
+export const BlueJeansDialInMessage = (props: DialInMessageProps) => {
+    const phoneLinkUsa = <OneTouchDialLink {...props} />;
     return (
         <span>
-            Having problems with video? As a back-up, you can call {phoneLinkUsa} from the USA 
-            (or <a target="_blank" href="https://www.bluejeans.com/premium-numbers"> find your international number to call in from outside the USA</a>) 
-            from any phone and enter {props.meetingNumber}#.
+            Having problems with video? As a back-up, you can call {phoneLinkUsa} from the USA
+            (or <IntlTelephoneLink {...props} />)
+            from any phone and enter {props.meetingNumber ?? meetingNumberPlaceholder}#.
+            You are not a moderator, so you do not need a moderator passcode.
```

Comment:
We need to check whether hosts are BJ moderators, but I do not think they are. This is relevant because the same `DialInMessage` components are used on both the attendee and host sides (in the Join Info dialog).

Reply:
It looks like the Meeting's `assignee` goes into the API call as `POST /v1/user/{assignee_bj_user_id}/scheduled_meeting`, and it looks like that does make them a moderator, because if I create a BJ meeting that way and then visit my BJ dashboard (https://bluejeans.com/scheduling/) I can see the Moderator Passcode.

Reply:
It looks like the Meeting's assignee goes into the API call as POST /v1/user/{assignee_bj_user_id}/scheduled_meeting, and it looks like that does make them a moderator, because if I create a BJ meeting that way and then visit my BJ dashboard (https://bluejeans.com/scheduling/) I can see the Moderator Passcode.

Reply:
Darn. But it doesn't require either the host or attendee to enter a moderator passcode when you call in, does it? Maybe we could have language similar to Zoom's "You do not need a host key or participant ID." Something like, "You do not need a passcode to enter the meeting"?

Reply:
Right, it doesn't require a passcode. That language sounds fine.

Reply:
Done. I chose to use "join the meeting" instead.